shinydashboard/Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM rocker/shiny-verse:3.6.3
+ARG cert_dir
+
+COPY ${cert_dir}/ /usr/local/share/ca-certificates
```

Comment:
for some reason something related to copying certs is not failing gracefully. this is why travis is breaking. Also had an issue locally before I moved my certs directory to the correct place. here's the error:

```
Step 3/10 : COPY ${cert_dir}/ /usr/local/share/ca-certificates
ERROR: Service 'shinydashboard' failed to build: COPY failed: stat /var/lib/docker/tmp/docker-builder001983342/certs: no such file or directory
```